tests/nvmeof/test_ceph_nvmeof_gateway.py

```diff
@@ -0,0 +1,173 @@
+"""
+Test suite that verifies the deployment of Ceph NVMeoF Gateway
+ with supported entities like subsystems , etc.,
+
+"""
+import json
+
+from ceph.ceph import Ceph
+from ceph.nvmeof.gateway import Gateway, configure_spdk
+from ceph.nvmeof.initiator import Initiator
+from ceph.parallel import parallel
+from ceph.utils import get_node_by_id
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+from utility.utils import generate_unique_id, run_fio
+
+LOG = Log(__name__)
+
+
+def configure_subsystems(rbd, pool, gw, config):
+    """Configure Ceph-NVMEoF Subsystems."""
+    sub_nqn = config["nqn"]
+    gw.create_subsystem(sub_nqn, config["serial"])
```

Comment:
Can we have log info for creating subsystrem - Its easy to identify in the logs

Reply:
Updated.